web/modules/custom/portland/modules/portland_groups/portland_groups.module

```diff
@@ -236,6 +236,15 @@ function _portland_groups_synchronize_entity_group_ownership($entity)
           }
         }
       }
+
+      // If the English version is updated, update other translations' paths
```

Comment:
Should this new code be in the other if statement above on line 200, since we only want it to run if the primary group was changed? (or am I misunderstanding the purpose?)

Reply:
Yes, I remembered that the group module is doing a "double save" and that caused both branches of the IF-ELSE statement to be executed. I've updated the code.